### PR TITLE
Update kubernetes-api.md

### PR DESCRIPTION
**Pull request by jiaj12**
sync with en master

### CODE REVIEW

File: content/zh/docs/contribute/generate-ref-docs/kubernetes-api.md [hunk 1]
```diff
@@ -123,12 +123,12 @@ Go to `<rdocs-base>`, and open the `Makefile` for editing:
...
 * Set `K8S_WEBROOT` to `<web-base>`.
 * Set `K8S_RELEASE` to the minor version of the docs you want to build.
-  For example, if you want to build docs for Kubernetes 1.17, set `K8S_RELEASE` to 1.17.
+For example, if you want to build docs for Kubernetes 1.17, set `K8S_RELEASE` to 1.17.
 -->
 * 设置 `K8S_ROOT` 为 `<k8s-base>`.
 * 设置 `K8S_WEBROOT` 为 `<web-base>`.
 * 设置 `K8S_RELEASE` 为要构建的文档的版本。
-  例如，如果您想为 Kubernetes 1.17 构建文档，请将 `K8S_RELEASE` 设置为 1.17。
+例如，如果您想为 Kubernetes 1.17 构建文档，请将 `K8S_RELEASE` 设置为 1.17。
```
Comment (tengqm) [inline content/zh/docs/contribute/generate-ref-docs/kubernetes-api.md:131]: pls reverse this change.

**Comment by jiaj12** [comment]
resolved, thx for the review @tengqm

**Pull request merged**